prompt-api-playground/script.js

```diff
@@ -0,0 +1,98 @@
+import { marked } from 'https://cdn.jsdelivr.net/npm/marked@13.0.3/lib/marked.esm.js';
+import DOMPurify from 'https://cdn.jsdelivr.net/npm/dompurify@3.1.6/dist/purify.es.mjs';
```

Comment:
:heart: yes, I wonder how often developers are forgetting this as a best practice for rendering AI generated content and exposing their sites to attacks.

Reply:
Yeah, easy to forget…